components/calendar/calendar.js

```diff
@@ -54,49 +39,66 @@ export default class Calendar extends crsbinding.classes.BindableElement {
     }
 
     async load() {
+        this.#perspectiveElement = this.shadowRoot.querySelector("#perspectiveElement");
+        this.#perspectiveHandler = this.#viewLoaded.bind(this);
+        this.#perspectiveElement.addEventListener("view-loaded", this.#perspectiveHandler);
+
         const tplCell = this.shadowRoot.querySelector("#tplCell");
         await crsbinding.inflationManager.register("calendar-cell", tplCell);
 
         const tplYears = this.shadowRoot.querySelector("#tplYears");
         await crsbinding.inflationManager.register("calendar-years", tplYears);
 
-        this.#tabHandler = this.#tabNavigation.bind(this);
-        this.shadowRoot.addEventListener('keydown', this.#tabHandler);
+        this.#keyboardInputManager = new CalendarKeyboardInputManager(this);
+        this.#changeMonthHandler = this.#changeMonth.bind(this);
+        this.addEventListener("change-month", this.#changeMonthHandler);
```

Comment:
I attempted to use registerEvent and tested it, it seemed to cause memory leaks as it wasn't being cleaned by the bindable elements dispose, it could be something similar to what was happening with the bindable events.